nipype/interfaces/tests/test_base.py

```diff
@@ -655,17 +655,17 @@ def test_CommandLine_output():
     ci = nib.CommandLine(command='ls -l')
     ci.inputs.terminal_output = 'allatonce'
     res = ci.run()
-    yield assert_equal, res.runtime.merged, ''
+    yield assert_equal, text_type(res.runtime.merged), text_type('')
```

Comment:
This should work fine without `text_type`.  On Python 2:

```
In [78]: unicode('hi there') == 'hi there'
Out[78]: True
```

On Python 3, literal strings are unicode, so you're comparing unicode to unicode already.